src/biogeochem/VOCEmissionMod.F90

```diff
@@ -933,13 +890,28 @@ function get_gamma_T(t_veg240_in, t_veg24_in,t_veg_in, ct1_in, ct2_in, betaT_in,
     if ( (t_veg240_in > 0.0_r8) .and. (t_veg240_in < 1.e30_r8) ) then 
        ! topt and Eopt from eq 8 and 9:
        topt = co1 + (co2 * (t_veg240_in-tstd0))
-       Eopt = Ceo_in * exp (co4 * (t_veg24_in-tstd0)) * exp(co4 * (t_veg240_in -tstd0))
+       if ( (ivt_in == nbrdlf_dcd_brl_shrub) ) then  ! boreal-deciduous-shrub
+       ! coming from BEAR-oNS campaign willows results
+             Eopt = 7.9_r8 * exp (0.217_r8 * (t_veg24_in - tfrz - 24.0_r8))
+       else if ( (ivt_in == nc3_arctic_grass ) ) then  ! boreal-grass
+             Eopt = exp(0.12_r8 * (t_veg240_in - tfrz - 15._r8))
```

Comment:
Same comment for 0.12.

The next line uses "co4" instead of hardwired values, so I now wonder whether 0.12 could be co4_c3_arc_grass and 0.217 should be c04_dcd_brl_shrub.

Similarly, might 7.9 be Ceo_in_dcd_brl_shrub?

Reply:
0.12 is the empirical parameter for the emission factors of Arctic C3 grass in response to 10-day temperature change.